Rakefile

```diff
@@ -6,18 +6,23 @@ Rake.application.load_imports
 
 task('metrics:mutant').clear
 namespace :metrics do
-  task :mutant => :coverage do
-    success = Kernel.system(*%w[
+  task mutant: :coverage do
+    jobs = 4 if ENV.key?('CIRCLE_CI')
+
+    arguments = %w[
       bundle exec mutant
-      --zombie
-      --use rspec
+      --ignore-subject Mutant::Meta*
       --include lib
-      --require mutant
       --since HEAD~1
-      --ignore-subject Mutant::Meta*
-      --
-      Mutant*
-    ]) or fail 'Mutant task is not successful'
+      --require mutant
+      --use rspec
+      --zombie
+    ]
+    arguments.concat(%W[--jobs 4]) if ENV.key?('CIRCLE_CI')
```

Comment:
I simply do not like "C-Style" APIs where integers heavily get abused for control flow triggers. Maybe mutant could support `--jobs detect` or something instead that makes it super clear.